index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header id="pageHeader">
+      <h1> bbr-test</h1>
+    </header>
+    
+    
+    <section id="testContainer">
+      <header>bbr-test 시작하기</header>
+      
+      <article >
+        <header>
```

Comment:
여기는 header말고 다른 태그를 이용하는 편이 좋을 것 같아요. 한 섹션에 헤더가 2개라서, h1~6를 사용하는 건 어떨까요?

Reply:
aritcle 의 헤더라 괜찮을 것 같다고 생각했었어요..! 그럼 h2로 수정할까요?!
20번줄~31번 줄의 아티클이 문제 수만큼 써지게 될것 같아요!

Reply:
음, 질문 제목이 정확히 어떤걸 표현하는 걸까요? 질문만 적게 된다면 p만 사용하면 될 거 같고.. 만약 제가 생각한 mbti 사이트 방식으로 질문 - 버튼 - 질문 - 버튼 방식으로 진행되는거라면 음... 제가 생각하는 article은 그 내용만 똑 떼서 봐도 뭔 지 알 수 있는 컨텐츠를 담는 태그라고 생각을 해서 차라리 결과 페이지에 사용하시는 것이 더 맞지 않을까 생각해요. 현재 페이지는 설문지니까 폼 관련 태그 이용하시는 건 어떨까요? label,  checkbox 등등.. 그리고 얘네는 묶어야 스타일 적용이 쉬울 것 같다 하는 애들은 div나 span 쓰셔도 괜찮아요.

Reply:
아! 이해했습니다.질문제목+ 질문 이렇게 있어야 생각했었는데, 잘못 생각했었네요..! 
```suggestion
  <article>
    <form>
      <label> 질문 </label>
      <div>
        <button></button>
        <button></button>
        <button></butt
        on>
    </form>
  </ariticle>
```

Reply:
내일 확인해주시면 한번에 수정해서 커밋하겠습니다:)

Reply:
둘 다 괜찮은 것 같아요 :)  고생하셨습니다ㅎㅎ 알바 다녀와서 저도 마저 하겠습니다